apps/maestro/src/features/ManageInterchainToken/actions/mint/MintInterchainToken.tsx

```diff
@@ -47,21 +50,24 @@ export const MintInterchainToken: FC = () => {
 
     const adjustedAmount = parseUnits(
       data.amountToMint,
-      erc20Details?.decimals || 18
+      erc20Details?.decimals ?? 18
```

Comment:
Curious what's the reason behind this change, wouldn't 0 decimals indicate a wrong value, so isn't a fallback to 18 preferred?